pkgs/desktops/gnome-3/core/tracker-miners/default.nix

```diff
@@ -51,6 +84,18 @@ in stdenv.mkDerivation rec {
     };
   };
 
+  postPatch = ''
+    substituteInPlace src/tracker-extract/10-flac.rule --replace audio/x-flac audio/flac
```

Comment:
Cool, is this why GNOME Music does not see my FLAC music?

Reply:
Could you add some comments to each of this replacements so we know why it is needed.

Reply:
Apparently, this was done to prioritize GStreamer extractor: https://git.gnome.org/browse/tracker-miners/commit/?id=b830fdf96a97e6f37a25a4bb58106748c585d8e3